website/src/hooks/useSubmissionOperations.ts

```diff
@@ -25,8 +27,11 @@ export function useSubmissionOperations(
     pageQuery: PageQuery,
 ) {
     const hooks = useMemo(() => backendClientHooks(clientConfig), [clientConfig]);
-    const allRelevantStatuses = [receivedStatus, inProcessingStatus, hasErrorsStatus, awaitingApprovalStatus];
+    const allRelevantStatuses = [receivedStatus, inProcessingStatus, processedStatus];
+    const allProcessingResults = [noIssuesProcessingResult, warningsProcessingResult, errorsProcessingResult];
     const [includedStatuses, setIncludedStatuses] = useState<string[]>(allRelevantStatuses);
+    const [includedProcessingResults, setIncludedProcessingResults] = useState<string[]>(allProcessingResults);
+    // TODO here I should add the include/exclude warnings thing
```

Comment:
Is this still a todo or just needs deleting?

Reply:
just needed deleting, thanks!

https://github.com/loculus-project/loculus/pull/3125/commits/01a6a5db34ea0dca22eb32aabfea5793d4d385b3